README.md

```diff
@@ -31,10 +31,14 @@ We welcome you to try it and look forward to feedback on the library! However, p
 
 ## Installation
 
-The easiest way to install OpenDP is using `pip` (the [package installer for Python](https://pypi.org/project/pip/)):
+Install OpenDP for Python with `pip` (the [package installer for Python](https://pypi.org/project/pip/)):
 
     $ pip install opendp
 
+Install OpenDP for R from an R session:
+
+    install.packages("opendp", repos = "https://opendp.r-universe.dev")
+
```

Comment:
Won't be true until we sort out repo weirdness.

Reply:
Goodbye GMP/MPFR, hello R-universe.